src/openapi/transport/putPatchDiagnosticsQueue.ts

```diff
@@ -0,0 +1,113 @@
+import log from '../../log';
+import TransportQueue from './queue';
+import type TransportCore from './core';
+import type {
+    HTTPMethodType,
+    HTTPMethod,
+    StringTemplateArgs,
+    RequestOptions,
+} from '../../types';
+import type { ITransport } from './transport-base';
+
+// fix-me typo

```

Comment:
Its ok to just fix I think

Reply:
Accidentally fixed it while I was doing a global replacement - removed the comment